### ISSUE TITLE
Orderby `unresolved` first and `resolved` last

### ISSUE THREAD

**Issue by mayeenulislam** [nanodesigns/nanosupport#22]
### Actual behavior

User @wptech68 [reported](https://wordpress.org/support/topic/some-problems-with-layout-and-notifications/) a feature request:
If you get to the `/support-desk/`, _you_ will get all the tickets submitted by _you_. But they are ordered by date only. If any ticket is resolved they are not ordered duly.

### Steps to reproduce

1. Create three tickets
2. Mark the second one resolved
3. See the ticket order in `/support-desk/`

### Expected behavior

The `resolved` tickets should be in least order prior to `open`, `pending`, and `under inspection` tickets.

### System Status

> NanoSupport: v0.3.0

**Comment by mayeenulislam** [comment]
It's good to implement such ordering. But...

- Is there anything we need to consider before proceeding developing such feature?
- Don't we need a **dropdown** orderby field so that the user can control which order they like?

**Comment by wptech68** [comment]
I agree with a dropdown where admin can set how tickets are ordered, but just the simple idea that the last ticket replied should be placed at top is the expected behaivour I guess

**Comment by mayeenulislam** [comment]
The possible sorting options are noted below:
```
+------------------------------+
| Default Sorting            v |
+------------------------------+
| Default Sorting              |
| Order by Resolved First      |
| Order by Open First          |
+------------------------------+
```
If you have anything other than this in mind, please share

**Comment by wptech68** [comment]
+------------------------------+
| Default Sorting            
+------------------------------+
| Default Sorting            
| Order by Resolved First
| Order by Open First     
| Order by Updated date
+------------------------------+

**Comment by wptech68** [comment]
any update on this feature?
currently, Im now with 170 tickets in 17 pages (10 per page) that are showing whatever in the first page, including resolved tickets , is hard to navigate looking for a specific ticket.
Im now thinking to remove definitely the resolved tickets to stop them showing, but that is not the correct behaivour i guess :)

**Comment by mayeenulislam** [comment]
I'm afraid, no update just now.
But we will prioritize this thread so that we can move for it ASAP.

**Comment by mayeenulislam** [comment]
Filtering Tickets in admin panel using the following filters activated in version `0.5.0`

- Ticket Status
- Ticket Priority
- Ticket Agent
